22.RaytracedAO/main.cpp

```diff
@@ -717,8 +725,24 @@ int main(int argc, char** argv)
 		return true;
 	};
 
+	auto processSensorsBehaviour = cmdHandler.getProcessSensorsBehaviour();
+	uint32_t sensorIndex = cmdHandler.getSensorID();
+	if (sensorIndex >= globalMeta->m_global.m_sensors.size())
+	{
+		printf("[WARNING]: A valid sensor ID was not found. Selecting the first sensor.\n");
+		sensorIndex = 0;
+	}
+
+	// Add the found sensor to sensor data, if no (or invalid) ID was specified then just add the first sensor.
+	const auto& sensor = globalMeta->m_global.m_sensors[sensorIndex];
+	extractAndAddToSensorData(sensor, sensorIndex);
```

Comment:
see the comment about adjusting iteration instead.

Reply:
Its already resolved, btw.